public/img/icons/unicons/legend-show.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="48" height="48" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
please see the guidelines [here](https://github.com/grafana/grafana/blob/main/public/img/icons/README.md) for how to properly format these icons. in particular, i think you'll need to remove the explicit `width` and `height` attributes